src/content/general/architecture/capi/index.md

```diff
@@ -0,0 +1,227 @@
+---
+linkTitle: Cluster API architecture
+title: The Giant Swarm Platform Architecture
+description: Architecture overview explaining how our Platform is built and what services do we offer.
+weight: 30
+menu:
+  main:
+    parent: general-architecture
+last_review_date: 2022-09-12
+user_questions:
+  - Do you run a Developer Platform?
```

Comment:
This user question is very general. I suggest something specific to Cluster API.